server/databasehelper.js

```diff
@@ -1,21 +1,36 @@
 import { Sequelize, DataTypes } from "sequelize";
+import { DATABASE, USERNAME, PASSWORD } from "./server.js";
 
 class databaseHelper {
-  constructor() {
-    // constructor(database, username, password) {
-    //   this.DATABASE = database;
-    //   this.USERNAME = username;
-    //   this.PASSWORD = password;
+  constructor(environment) {
+    if (environment == "development") {
+      this.DATABASE = DATABASE;
+      this.USERNAME = USERNAME;
+      this.PASSWORD = PASSWORD;
 
-    // this.db = new Sequelize(this.DATABASE, this.USERNAME, this.PASSWORD, {
-    //   host: "localhost",
-    //   dialect: "postgres",
-    // });
+      this.db = new Sequelize(this.DATABASE, this.USERNAME, this.PASSWORD, {
+        host: "localhost",
+        dialect: "postgres",
+      });
+    } else {
+      this.db = new Sequelize(
+        "postgresql://postgres:TcsqgayINAiUgfGjBdzcfLHOVAqukiZH@postgres-4tco.railway.internal:5432/railway"
```

Comment:
You probably don't want to check this URL (and password) into source code control. Instead, but it in an env variable (I think it might already be set in `DATABASE_URL`, so you can use `process.env.DATABASE_URL`).

Reply:
Fixed! I'm unsure how to remove this from my version history, though, I will put that in our rolling agenda for Monday as well.

Reply:
Added to issues